core/modules/system/system.module

```diff
@@ -937,34 +911,89 @@ function system_menu() {
     'file' => 'system.admin.inc',
     'weight' => 20,
   );
+
+  // URL handling.
+  $items['admin/config/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Settings related to URLs.',
+    'position' => 'right',
+    'weight' => -20,
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => array('access administration pages'),
+    'file' => 'system.admin.inc',
+  );
+  $items['admin/config/urls/urls'] = array(
+    'title' => 'URL settings',
```

Comment:
`admin/config/urls/urls` :frowning: 

Since this page is "URL settings" why not `admin/config/urls/settings`?